src/components/building-information/building-information.component.tsx

```diff
@@ -48,12 +50,15 @@ class BuildingInformation extends React.Component<IProps, IState> {
       showBuildingInfo
     } = this.props;
     const draggableRange = { top: height / 2, bottom: 105 };
+    const { allowDragging } = this.state;
 
     return (
       <SlidingUpPanel
-        allowDragging={this.state.allowDragging}
+        allowDragging={allowDragging}
         draggableRange={draggableRange}
+        /*eslint-disable */
```

Comment:
having comments like this is discouraged, its better to modify the actual config rather than putting comments like these.